oligo_designer_toolsuite/oligo_specificity_filter/_filter_bowtie.py

```diff
@@ -0,0 +1,337 @@
+import os
+import re
+import time
+from pathlib import Path
+from subprocess import Popen
+
+import pandas as pd
+from joblib import Parallel, delayed, parallel_backend
+
+from ._filter_base import ProbeFilterBase
+from ._utils import (
+    _load_probes_info,
+    _write_output,
+    _write_removed_genes,
+    mismatch_in_ligation,
+)
+
+
+class ProbeFilterBowtie(ProbeFilterBase):
+    def __init__(
+        self,
+        n_jobs,
+        dir_output,
+        dir_annotations,
+        file_transcriptome_fasta,
+        min_mismatches=4,
+        mismatch_region=None,
+        ligation_region=0,
```

Comment:
Here, again we should merge ligation_site and ligation_region variables into seed_Region, which is defined as tuple (see blast filter)